cmd/wol/wol.go

```diff
@@ -167,13 +136,12 @@ func wakeCmd(args []string, aliases *Aliases) error {
 	}
 	defer conn.Close()
 
-	fmt.Printf("Attempting to send a magic packet to MAC %s\n", macAddr)
-	fmt.Printf("... Broadcasting to: %s\n", bcastAddr)
-	n, err := conn.Write(bs)
-	if err == nil && n != 102 {
-		err = fmt.Errorf("magic packet sent was %d bytes (expected 102 bytes sent)", n)
-	}
-	if err != nil {
+	w := wol.New(
```

Comment:
Why separate the New() from the Wake()? Is this so you can keep an instance and wake up the same dude again and again? Seems like having the interface be `wol.Wake(...)` encompass the `New` stuff may by nicer? Thoughts?

Reply:
Valid point. That's basically it, to allow the instance to be reuseable. The idea behind the `New()` was to also provide a nice way to initialise the struct. I've found this pattern quite common in many big Go projects and I ended up following it. It's not that you cannot initialise the struct directly, I just found more go-like to have a `New()` function available.

Reply:
If you're ok with this, I can move forward and port the latest change you've from https://github.com/sabhiram/go-wol/commit/d6d48113fcacd37de2484f613b775eac8ec2cdcc before merging. :)